lesson_2/test5.py

```diff
@@ -0,0 +1,28 @@
+# 5. a function that takes two lists of numbers and returns the result of merging the lists. 
+# The elements of the first list should become the elements at the even indexes of the returned list, 
+# while the elements of the second list should become the elements at the odd indexes. 
+# For instance: merge([1, 2, 3], [4, 5, 6]) # => [1, 4, 2, 5, 3, 6]
+
+list1 = [1, 2, 3]
+list2 = [4, 5, 6]
+# GET 2 list of ints
+def merge(list1, list2):
+    # SET/READ zip_list --> use zip to merge lists
+    zip_list = zip(list1, list2)
+    # SET/READ new_list lists merged as list of tuples
+    new_list = list(zip_list)
+    # SET merge_list = to append values of tuple
+    merge_list = []
+    # SET tuple1, tuple2 to store values from in tuple
+    tuple1 = 0
+    tuple2 = 0
+    # WHILE for items in new_list index items from tuple and append to merge_list
+    for element in new_list:
+        tuple1 = element[0]
```

Comment:
You don't need to explicitly create variables, you can loop through a list of tuples like this:

```suggestion

    # WHILE for items in new_list index items from tuple and append to merge_list
    for element1, element2 in new_list:
        merge_list.append(element1)
        merge_list.append(element2)
```

Reply:
good to know